bert-wsd-vis/deploy.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+#!/usr/bin/env bash
+# Copyright 2018 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+
+# This script deploys the demo to GCP.
+# Add --upload_jsons to copy the json files as well.
+
+
+echo "Building..."
+yarn
+yarn build
+
+echo "Deploying..."
+gsutil mkdir -p gs://bert-wsd-vis/demo
+gsutil -m cp static/* gs://bert-wsd-vis/demo
+
+gsutil -m setmeta -h "Cache-Control:private" "gs://bert-wsd-vis/**.html"
+gsutil -m setmeta -h "Cache-Control:private" "gs://bert-wsd-vis/**.css"
+gsutil -m setmeta -h "Cache-Control:private" "gs://bert-wsd-vis/**.js"
+
+if [[ $* == *--upload_jsons* ]]; then
+  echo 'Uploading jsons data'
+  gsutil mkdir -p gs://bert-wsd-vis/demo/jsons
+  gsutil -m cp static/jsons/* gs://bert-wsd-vis/demo/jsons
+fi
+
+echo "The demo is now available at https://storage.googleapis.com/bert-wsd-vis/demo/index.html?#"
```

Comment:
Cool!  Thanks for sharing this awesome work! 😄 

These are super interesting to explore, and the way that the detail of labels adapts as you pan and zoom around is awesome!  It makes it super quick to explore around and see the complexity of the senses within the top-level clusters too (eg, "die" in terms of processor fabrication or something like that :)).

I was fiddling around trying to reproduce some things from the paper like this:
<img width="722" alt="Screen Shot 2019-07-26 at 12 08 02 PM" src="https://user-images.githubusercontent.com/1056957/61965381-09770780-af9e-11e9-9c48-18fb85155586.png">

and saw this:

<img width="989" alt="Screen Shot 2019-07-26 at 12 13 44 PM" src="https://user-images.githubusercontent.com/1056957/61965699-d5e8ad00-af9e-11e9-82ff-3bd98edb8e16.png">

It does looks like this separates out the articles with German into their own cluster, but there's English mixed in there too, and it's a bit different than what the paper shows.  I think I was figuring this was the release of the tool you all used to make the visuals in the paper, but I'm wondering if there are other differences than what's described?

I see this is the first step of open sourcing, so apologies if this is distracting as you're working, I'm just being curious and excited.  Thanks for sharing this awesome work! 👍